<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>This file is part of the xTuple ERP: PostBooks Edition, a free and
open source Enterprise Resource Planning software suite,
Copyright (c) 1999-2014 by OpenMFG LLC, d/b/a xTuple.
It is licensed to you under the Common Public Attribution License
version 1.0, the full text of which (including xTuple-specific Exhibits)
is available at www.xtuple.com/CPAL.  By using this software, you agree
to be bound by its terms.</comment>
 <class>characteristicAssignment</class>
 <widget class="QDialog" name="characteristicAssignment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>366</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item Characteristic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="_characteristicLit">
       <property name="text">
        <string>Characteristic:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="_char"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="_valueLit">
       <property name="text">
        <string>&amp;Value:</string>
       </property>
       <property name="buddy">
        <cstring>_value</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QStackedWidget" name="_stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="_textPage">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="XLineEdit" name="_value"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="_listPage">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="XComboBox" name="_listValue"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="_datePage">
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="DLineEdit" name="_dateValue"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="_listpriceLit">
       <property name="text">
        <string>&amp;List Price:</string>
       </property>
       <property name="buddy">
        <cstring>_value</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="XLineEdit" name="_listprice"/>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="_default">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLineEdit</class>
   <extends>QLineEdit</extends>
   <header>xlineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_value</tabstop>
  <tabstop>_listprice</tabstop>
  <tabstop>_default</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
